polaris-react/src/components/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -109,10 +125,11 @@ export function VideoThumbnail({
   }
 
   return (
-    <div
-      className={styles.Thumbnail}
-      style={{backgroundImage: `url(${thumbnailUrl})`}}
-    >
+    <div className={styles.ThumbnailContainer}>
+      <div
+        className={styles.Thumbnail}
+        style={{backgroundImage: `url(${thumbnailUrl})`}}
+      />
```

Comment:
separated out the background image from the wrapping div so that the image overflow wouldn't hide the play buttons when the image cover had to be cropped. For example, on mobile:

<img width="262" alt="Screen Shot 2022-10-04 at 10 20 14 AM" src="https://user-images.githubusercontent.com/20652326/193848885-d4860b45-dcb4-45de-84cb-b899632ca775.png">
<img width="256" alt="Screen Shot 2022-10-04 at 10 20 32 AM" src="https://user-images.githubusercontent.com/20652326/193848919-5aa6f2a6-a5e4-4c8e-87ec-222e2988d2e1.png">